onnxruntime/contrib_ops/cuda/quantization/dequantize_blockwise_bnb4.cuh

```diff
@@ -11,6 +11,38 @@ namespace cuda {
 template <class T>
 Status SetBnbQuantMap(int quant_type, T* quant_map_buffer, cudaStream_t stream);
 
+// templated scalar multiply function
+template <class T>
+__device__ inline T ScalarMul(T a, T b);
```

Comment:
Just saw that the architecture checks are already handled in https://github.com/microsoft/onnxruntime/blob/a5537f2f563d4975c7e6121a7eb260bbbfd9455a/onnxruntime/core/providers/cuda/cu_inc/common.cuh

Will remove the templated functions and try with normal operators directly.

Reply:
Done